reframe/core/logging.py

```diff
@@ -197,6 +197,26 @@ def _create_filelog_handler(handler_config):
     return MultiFileHandler(filename_patt, mode='a+' if append else 'w+')
 
 
+def _create_syslog_handler(handler_config):
+    from socket import SOCK_DGRAM, SOCK_STREAM
+    address = handler_config.get('address', None) 
+    facility = handler_config.get('facility', logging.handlers.SysLogHandler.LOG_USER)
```

Comment:
The only problem with my proposed solution above is that there is that `SysLogHandler` doesn't test the validity of the facility, and if it's wrong it will throw an error during logging, which is not what we want; not so user friendly. So I think it's better if you do the following:

```python
try:
    facility_type = logging.handlers.SysLogHandler.facility_names[facility]
except KeyError:
    raise ConfigError('syslog handler: unknown facility: %s' % facility) from None
```

Reply:
Thank you for the suggestion. I have changed the code accordingly.